pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1490,4 +1489,41 @@ self: super: {
   x509-validation = dontCheck super.x509-validation;
   tls = dontCheck super.tls;
 
+  # Bump requested on upstream https://github.com/bgamari/monoidal-containers/pull/62
+  monoidal-containers = overrideCabal super.monoidal-containers (drv: {
+    postPatch = (drv.postPatch or "") + ''
+      substituteInPlace monoidal-containers.cabal --replace "these >= 1 && <1.1" "these -any"
+    '';
+  });
+
+  # Fixing test and deps on upstream on https://github.com/reflex-frp/patch/pull/17
+  patch = dontCheck (doJailbreak (overrideCabal (super.patch.override {
+    dependent-map = super.dependent-map_0_3_1_0;
+  }) (drv: {
+    postPatch = (drv.postPatch or "") + ''
+      substituteInPlace patch.cabal --replace "these >=1 && <1.1" "these -any"
+    '';
+  })));
+
+  # Bumped bounds in upstream PR https://github.com/reflex-frp/reflex/pull/434
+  # Tests disabled because of very old dep: https://github.com/reflex-frp/reflex/issues/435
+  reflex = dontCheck (doJailbreak (overrideCabal (super.reflex.override {
+    dependent-map = super.dependent-map_0_3_1_0;
+  }) (drv: {
+    postPatch = (drv.postPatch or "") + ''
+      substituteInPlace reflex.cabal --replace "haskell-src-exts >=1.16 && <1.23" "haskell-src-exts -any"
+      substituteInPlace reflex.cabal --replace "these >=1 && <1.1" "these -any"
+      substituteInPlace reflex.cabal --replace "dependent-sum >=0.6 && <0.7" "dependent-sum -any"
+    '';
+  })));
+
+  # Fixed bounds in upstream PR https://github.com/reflex-frp/reflex-dom/pull/388
+  # Tests disabled and broken overrided needed because of missing lib chrome-test-utils.
+  reflex-dom-core = unmarkBroken (dontCheck (doJailbreak (overrideCabal (super.reflex-dom-core.override {
```

Comment:
Another couple possibilities:

- Bug upstream to release `chrome-test-utils` to Hackage.  (It is frustrating when people release libraries to Hackage that have dependencies that aren't on Hackage.  It is also somewhat frustrating that Hackage allows this.)

- Send a PR to `cabal2nix` teaching it about this special case for `reflex-dom-core`, where tests shouldn't be run, and it shouldn't be marked broken.  (Although I don't know if this will work well, since there could be multiple reasons for it to be marked broken.)

Reply:
I am bugging upstream about it. Do you consider this a merge blocker in between?
Special casing this in cabal2nix sounds very ugly to me.